lab_1_keywords_tfidf/main.py

```diff
@@ -66,10 +88,20 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(frequencies, dict) and isinstance(top, int):
+        top_dic = {}
+        top_values = sorted(frequencies.values(), reverse=True)
+        top_n_values = top_values[:top]
+        for i in top_n_values:
+            for k in frequencies.keys():
+                if frequencies[k] == i:
```

Comment:
what if more than one word have the same frequency?

Reply:
the word which is read by the program earlier (which is located closer to the beginning of the dictionary) returns before another word with the same frequency

Reply:
is this the behaviour that we want? 
don't we need to collect all the keys values of which is high enough regardless of where in the dict this key is located?

Reply:
I don't understand. It returns both keys if their values are the same, but if i choose top = 8, for example, and two keys, which are supposed to be the last in the top (8th), have the same frequency, it returns only one of them, whichever it reads the first.